tekton/src/pipelines/taskdefs/fvt-manage/phase5.yml.j2

```diff
@@ -25,3 +25,27 @@
     - fvt-manage-directprint
     - fvt-manage-adhoc-report
     - fvt-manage-jms-outbound
+
+# Manage FVT toolsapi
+- name: fvt-manage-toolsapi
+  {{ lookup('template', 'taskdefs/fvt-manage/api/taskref.yml.j2') | indent(2) }}
+  params:
+    {{ lookup('template', 'taskdefs/fvt-manage/api/params.yml.j2') | indent(4) }}
+    - name: fvt_test_suite_prefix
+      value: base
+    - name: fvt_test_suite
+      value: toolsapi
+  runAfter:
+    - fvt-manage-setup
+
+# Manage FVT esig
+- name: fvt-manage-esig
```

Comment:
Can we please name it something else? Selenium tests are also part of phase 5 and its called "fvt-manage-esig-basic", so it would be best if we can also add some meaningful suffix after esig which explain its working.

Reply:
Thank you. I have given different logical name.